content/about/about-content.md

```diff
@@ -28,8 +28,12 @@ A lot of the work in EEW stems from EDGI’s work around “Environmental Data J
 NAMES ETC
 
 ## Collaborators
-![EEW's collaborators: Stony Brook University Geospatial Center, Teaching Learning Lab, Sunrise Boston](./collabs.png)
+<a href="https://www.stonybrook.edu/commcms/gss/" target=_blank >Geospatial Center at Stony Brook University</a>
```

Comment:
In Markdown, you have to hit enter twice to get things on different lines – currently these will all render on one line (you can see by clicking 
![Screen Shot 2020-10-02 at 1 39 11 PM](https://user-images.githubusercontent.com/454690/94967805-aa235a00-04b4-11eb-8d11-2c38fd4fcd8e.png)
in the "Files changed" tab)

To fix: either put an extra newline between each line, or use `*` or `-` to turn it into a bulleted list